Explorer/Assets/DCL/Chat/ChatCommandsHandler.cs

```diff
@@ -36,5 +36,8 @@ public bool TryGetChatCommand(in string message, ref (IChatCommand command, Matc
 
             return false;
         }
+
+        public bool StartsLikeCommand(string message) =>
+            message.TrimStart()!.StartsWith(CHAT_COMMAND_CHAR);
```

Comment:
this will result in allocation for a new string. Do we really need a `Trim` for commands?